cypress/integration/homepage_spec.js

```diff
@@ -44,12 +45,36 @@ describe('moviesReq User Flows', () => {
         cy.contains('Money Plane')
         cy.contains(`It's a movie!`)
         cy.contains('Bad')
-        cy.contains('6.666666666666667')
-        cy.contains('2020-09-29')
+        cy.contains('6.67')
+        cy.contains('$63,000,000')
+        cy.contains('$100,853,753')
+        cy.contains('09-29-2020')
         cy.get('.genre-container').should('include.text', 'Drama')
         cy.contains('Drama')
       });
 
+    it('Should not show budget or revenue if those values are 0', () => {
+        cy.intercept('GET', moviesReq, {
+            statusCode: 200,
+            fixture: 'movies'
+        })
+        cy.intercept('GET', singleMovieReq2, {
+            statusCode: 200,
+            fixture: 'movie2'
+        }) 
+        .visit('http://localhost:3000')
+        .get('a[href="/movies/718444"]')
+        .click()
+        .url().should('include', '/movies/718444')
+        cy.contains('Rogue')
+        cy.contains(`When the hunter becomes the prey.`)
+        cy.contains('7.00')
```

Comment:
Because your test above this already confirms that the tagline, title, release date, etc. does render to the page, it's redundant to check for those things in this test. I would say that the only thing you do need to test in this scenario is that the budget and revenue do not render to the page.